README.adoc

```diff
@@ -118,38 +118,40 @@ The `@Type(type="uuid-char")` annotation specifies that this UUID value will be
 Leaving out this annotation causes a `BYTES` column to be used.
 
 ==== Use @GeneratedValue for ID Generation
-
 __NOTE__: Read to the end of this section to see the recommended way to set up `@GeneratedValue`.
 
-Hibernate's `@GeneratedValue` annotation for numeric fields is supported, and will by default use a
-positive bit-reversed sequence. A bit-reversed sequence internally uses a monotonically increasing
-counter  that is reversed before being returned to Hibernate. This means that the identifiers that
-are generated are in the form bitReversePositive(1), bitReversePositive(2), ...:
-[source, java]
+❌ Not recommended
+
+`GenerationType.AUTO`, `GenerationType.TABLE`, `GenerationType.SEQUENCE` - Not supported by the Spanner dialect.
```

Comment:
nit:

```suggestion
`GenerationType.AUTO`, `GenerationType.TABLE`, `GenerationType.SEQUENCE`: These are not supported by the Spanner dialect.
```

Note for the longer term (so not for this PR):
1. We should support AUTO. That should lead to the 'least-bad' auto-generation strategy that we can support, which I believe with our current feature set would be IDENTITY.
2. We should support TABLE if possible. I believe that the current limitation is that Hibernate generates an initialization command that tries to insert a row without specifying the column name. We should try to either override that in our dialect, or change this in Hibernate itself.